subprojects/scala/src/integTest/groovy/org/gradle/scala/compile/IncrementalScalaCompileIntegrationTest.groovy

```diff
@@ -53,13 +48,10 @@ class IncrementalScalaCompileIntegrationTest extends AbstractIntegrationSpec {
         file('src/main/scala/IPerson.scala').assertIsFile().copyFrom(file('NewIPerson.scala'))
 
         then:
-        // This gets reset each time you run() is run.
-        executer.expectDeprecationWarning();
         runAndFail("classes").assertHasDescription("Execution failed for task ':compileScala'.")
     }
 
     @Issue("GRADLE-2548")
-    @Ignore
```

Comment:
No idea. It did work before I changed anything, though. Seems like a fix in the zinc compiler.